.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+// prettier.config.js or .prettierrc.js
+module.exports = {
+  trailingComma: "es5",
+  tabWidth: 2,
+  semi: true,
+  singleQuote: false,
+};
```

Comment:
Config file for vscode extension.. not relevant to the PR, but I needed it so that I didn't screw up the stying